gateways.txt

```diff
@@ -41,40 +36,26 @@ https://crustwebsites.net
 https://ipfs0.sjc.cloudsigma.com
 http://ipfs.genenetwork.org
 https://ipfs.eth.aragon.network
-https://ipfs.smartholdem.io
 https://ipfs.xoqq.ch
-http://natoboram.mynetgear.com:8080
 https://video.oneloveipfs.com
 https://ipfs.scalaproject.io
-https://search.ipfsgate.com
 https://ipfs.decoo.io
-https://alexdav.id
-https://ipfs.uploads.nu
 https://hub.textile.io
-https://ipfs1.pixura.io
 https://ravencoinipfs-gateway.com
 https://ipfs.tubby.cloud
 https://ipfs.lain.la
 https://ipfs.kaleido.art
-https://ipfs.slang.cx
 https://ipfs.arching-kaos.com
 https://storry.tv
 https://ipfs.1-2.dev
 https://dweb.eu.org
 https://permaweb.eu.org
 https://ipfs.namebase.io
-https://ipfs.tribecap.co
-https://ipfs.kinematiks.com
 https://nftstorage.link
-https://gravity.jup.io
-http://fzdqwfb5ml56oadins5jpuhe6ki6bk33umri35p5kt2tue4fpws5efid.onion
```

Comment:
@2color fwiw http://en.wikipedia-on-ipfs.org.ipns.fzdqwfb5ml56oadins5jpuhe6ki6bk33umri35p5kt2tue4fpws5efid.onion/wiki/ works fine in Tor-enabled browser. 

I've added code to skip `.onion` ones for now.